docs/documentation.md

```diff
@@ -46,46 +62,127 @@ All request should have in URL `stationId` param encoded with Base64.
     }
     ```
     
+   - Response
+
+   ```json
+    {
+      "id": "xxxxxxxxxxxxxxxxxxxx",
+      "title": "Task Two",
+      "description": "This is the second task",
+      "priority": "MEDIUM",
+      "isDone": False,
+      "date": "2023-11-15",
+      "notificationTime": "10:00:00"
+    }
+    ``
+    
 - `PATCH /tasks/{taskId}` - Update a specific task.
     - Request
+
+    ```json
+    {
+      "title": "new task",
+      "description": "This is the new task",
+      "priority": "HIGH",
+      "isDone": true,
+      "date": "2022-12-23",
+      "notificationTime": "12:04:32" // optional
+    }
+    ```
+
+    - Response
     
     ```json
     {
-      "title": "Updated Task",
-      "description": "Updated details about the task",
-      "isDone": false,
-      "priority": "MEDIUM",
-      "date": "2023-11-15"
+      "id": "xxxxxxxxxxxxxxxxxxxx",
+      "title": "new task",
+      "description": "This is the new task", 
+      "priority": "HIGH",
+      "isDone": true,
+      "date": "2022-12-23",
+      "notificationTime": "12:04:32"
+    }
+    ```
+    
+- `PATCH /tasks/{taskId}` - Update a specific task.
+    - Request
+
+    ```json
+    {
+      "title": "Updated task2", // optional
+      "description": "This is the Updated task", // optional
+      "priority": "HIGH", // optional
+      "isDone": true, // optional
+      "date": "2022-12-23", // optional
+      "notificationTime": "12:04:32" // optional
+    }
+    ```
+
+    - Response
+    
+    ```json
+    {
+      "id": "xxxxxxxxxxxxxxxxxxxx",
+      "title": "Updated task2",
+      "description": "This is the Updated task",
+      "priority": "HIGH",
+      "isDone": true,
+      "date": "2022-12-23",
+      "notificationTime": "12:04:32"
     }
     ```
     
 - `DELETE /tasks/{taskId}` - Delete a specific task.
+    - Response:
+
+    ```json
+    {
+      "id": "xxxxxxxxxxxxxxxxxxxx",
+      "title": "Updated task2",
+      "description": "This is the Updated task",
+      "priority": "HIGH",
+      "isDone": true,
+      "date": "2022-12-23",
+      "notificationTime": "12:04:32"
+    }
+    ```
     
 **Statistics:**
 
-- `GET /statistics` - Get statistics for the authenticated user.
+- `GET /statistics` - Get statistics for the authenticated user for a specific date.
+    - Request
+
+    ```json
+    {
+      "Date": "2022-12-23"
+    }
+    ```
```

Comment:
Could we make request and responses in camelCase like we have other variables?

Reply:
surę